src/test/resources/examples/enumExamples/models/Models.kt

```diff
@@ -128,3 +135,49 @@ enum class ExtensibleEnumObject(
         fun fromValue(value: String): ExtensibleEnumObject? = mapping[value]
     }
 }
+
+data class FooBars(
+    @param:JsonProperty("prop_one")
+    @get:JsonProperty("prop_one")
+    val propOne: List<FooBarsFoo>? = null,
+    @param:JsonProperty("prop_two")
+    @get:JsonProperty("prop_two")
+    @get:Valid
+    val propTwo: List<FooBarsBar>? = null
+)
+
+data class FooBarsBar(
```

Comment:
**suggestion:** Would it make more sense to add some version of "Item" to the generated name? I feel like just concatenating schema name and property name has a higher potential for conflicts.

Reply:
It would be nice, but would be a larger refactor to make it play nice with Controller references